hack/lib/golang.sh

```diff
@@ -479,10 +481,11 @@ kube::golang::outfile_for_binary() {
   local binary=$1
   local platform=$2
   local output_path="${KUBE_GOPATH}/bin"
+  local bin
   if [[ "${platform}" != "${host_platform}" ]]; then
     output_path="${output_path}/${platform//\//_}"
   fi
-  local bin=$(basename "${binary}")
+  bin=$(basename "${binary}")
```

Comment:
nit: move this immediately under the `local bin`

Reply:
Thanks comment. I updated the line. Could you review it?